### ISSUE TITLE
Lazy loading chunks downloaded before route visiting

### ISSUE THREAD

**Issue by noscript** [vuejs/vue-cli#1153]
### Version
3.0.0-beta.6

### Reproduction link
[https://github.com/noscript/vue-cli-lazy-load](https://github.com/noscript/vue-cli-lazy-load)

### Steps to reproduce
Change About to an async component in `router.js`:
```diff
-import About from "./views/About.vue";
+const About = () => import("./views/About.vue");
```
Serve the app and open in browser. Check Networks tab in Chrome devtools, chunk `0.js` is downloaded with prefetch. Visit About view, `0.js` is downloaded normally.

### What is expected?
Chunk is downloaded only when visiting the route, according to https://router.vuejs.org/en/advanced/lazy-loading.html

### What is actually happening?
Chunks from all async components are downloaded at once.

---
This is a regression from `vue-cli 2.9.3`. Although the chunks are defined with `prefetch`, they are actually downloaded anyway (on the background, idle time etc).

**Comment by emanuelmutschlechner** [comment]
Can confirm. 

Tested it with 3 separate chunked `About.vue` pages and added a large image to `index.html`. 
```html
<head>
    <link rel="prefetch" as="image" href="image.png">
</head>
```
Results in Chrome:
![ac](https://user-images.githubusercontent.com/27734375/39020084-f35cc392-442b-11e8-97cb-723d9e8ecfa3.PNG)
Results in Firefox: 
![af](https://user-images.githubusercontent.com/27734375/39020088-f6318d82-442b-11e8-8717-0f7271b2e1a4.PNG)

Seems like Firefox starts downloading `prefetch` elements when document is ready, but Chrome not. I used network throttling for more accurate timeline
